kmir/src/kmir/cli.py

```diff
@@ -0,0 +1,293 @@
+    )
+    show_subparser.add_argument(
+        '--haskell-dir',
+        dest='haskell_dir',
+        type=dir_path,
+        help='Path to Haskell definition to use.',
+    )
+    show_subparser.add_argument(
+        '--spec-file',
+        dest='spec_file',
+        type=file_path,
+        help='Path to specification file',
+    )
+    show_subparser.add_argument(
+        '--save-directory',
+        dest='save_directory',
+        type=dir_path,
+        help='Path to KCFG proofs directory, directory must already exist.',
+    )
+
+    # KCFG view
+    view_subparser = command_parser.add_parser(
+        'view-kcfg',
+        help='Display tree view of CFG',
+        parents=[logging_args],
+    )
+
+    view_subparser.add_argument(
+        '--definition-dir',
+        dest='definition_dir',
+        type=dir_path,
+        help='Path to LLVM definition to use.',
+    )
+    view_subparser.add_argument(
+        '--haskell-dir',
+        dest='haskell_dir',
+        type=dir_path,
+        help='Path to Haskell definition to use.',
+    )
+    view_subparser.add_argument(
+        '--spec-file',
+        dest='spec_file',
+        type=file_path,
+        help='Path to specification file',
+    )
+    view_subparser.add_argument(
+        '--save-directory',
+        dest='save_directory',
+        type=dir_path,
+        help='Path to KCFG proofs directory, directory must already exist.',
+    )
+
+    return parser
+
+
+class KMIRCLIArgs(KCLIArgs):
+    @cached_property
+    def kcfg_show_args(self) -> ArgumentParser:
+        args = ArgumentParser(add_help=False)
+        args.add_argument(
+            '--node',
+            type=node_id_like,
+            dest='nodes',
+            default=[],
+            action='append',
+            help='List of nodes to display as well.',
+        )
+        args.add_argument(
+            '--node-delta',
+            type=arg_pair_of(node_id_like, node_id_like),
+            dest='node_deltas',
+            default=[],
+            action='append',
+            help='List of nodes to display delta for.',
+        )
+        args.add_argument(
+            '--failure-information',
+            dest='failure_info',
+            default=False,
+            action='store_true',
+            help='Show failure summary for cfg',
+        )
+        args.add_argument(
+            '--no-failure-information',
+            dest='failure_info',
+            action='store_false',
+            help='Do not show failure summary for cfg',
+        )
+        args.add_argument(
+            '--to-module', dest='to_module', default=False, action='store_true', help='Output edges as a K module.'
+        )
+        args.add_argument(
+            '--pending', dest='pending', default=False, action='store_true', help='Also display pending nodes'
+        )
+        args.add_argument(
+            '--failing', dest='failing', default=False, action='store_true', help='Also display failing nodes'
+        )
```

Comment:
argparse will infer precisely these names for you, but I presume that you're specifying the `dest` explicitly for consistency with the rest of the options

Reply:
(see other places as well for a similar comment)